AWS/aws-cloudtrail/tests/7602ff70-7e5f-42e9-86b2-36803df39183.json

```diff
@@ -25,29 +25,7 @@
     },
     "@timestamp": "2022-08-31T09:48:47Z",
     "action": {
-      "name": "ModifyInstanceAttribute",
```

Comment:
is this modification intended ? I cannot find the value duplicated in another field

Reply:
it's in event.action

Reply:
this field does not appear in the changelog

Reply:
i have to put every field ?

Reply:
Ah yes my bad

for the CHANGELOG.md its all good thank you